dataland-frontend/cypress.json

```diff
@@ -1,6 +1,6 @@
 {
   "pluginsFile": "tests/e2e/plugins/index.js",
-  "defaultCommandTimeout": 6000,
+  "defaultCommandTimeout": 480000,
```

Comment:
Dies bedeutet dass er wirklich bei jedem Attempt solange wartet bis er einen retry macht, auch bei einem Tooltip oder Button. Man kann auch in den Test files eine lokale configuration machen, so dass zB nur bei den request Dingen so ein langer TimeOut genutzt wird.

Reply:
Wurde angepasst und diskutiert